### ISSUE TITLE
Something went wrong error. (Looks like "Unable to read VR Path Registry" is the reason)

### ISSUE THREAD

**Issue by JFMugen** [Rainbox-dev/DuME#223]
I just tried to render a small project (10 seconds to render) but it failed. Just to be clear both today and yesterday, it worked once. (yesterday 3.rd try, today 1.st try) All other attempts are failed. 

AE Console Log;

```
Ready!
[17:50:48.891]: aerender version 15.1.2x69
aerender version 15.1.2x69
Using DXGI: Device: "NVIDIA GeForce GTX 960M" has dedicated video RAM (MB): 2010

[17:50:48.891]: aerender version 15.1.2x69
aerender version 15.1.2x69
Using DXGI: Device: "NVIDIA GeForce GTX 960M" has dedicated video RAM (MB): 2010

[17:50:49.219]: Using DXGI: Device: "NVIDIA GeForce GTX 960M" has dedicated video RAM (MB): 2010
Using DXGI: Device: "NVIDIA GeForce GTX 960M" has dedicated video RAM (MB): 2010

[17:50:49.220]: Using DXGI: Device: "NVIDIA GeForce GTX 960M" has dedicated video RAM (MB): 2010
Using DXGI: Device: "NVIDIA GeForce GTX 960M" has dedicated video RAM (MB): 2010

[17:50:50.539]: Using DXGI: Device: "NVIDIA GeForce GTX 960M" has dedicated video RAM (MB): 2010
Unable to read VR Path Registry from C:\Users\Gokhan\AppData\Local\openvr\openvrpaths.vrpath

[17:50:50.540]: Using DXGI: Device: "NVIDIA GeForce GTX 960M" has dedicated video RAM (MB): 2010
Unable to read VR Path Registry from C:\Users\Gokhan\AppData\Local\openvr\openvrpaths.vrpath

[17:50:51.241]: Unable to read VR Path Registry from C:\Users\Gokhan\AppData\Local\openvr\openvrpaths.vrpath
InitializeFileLogging failed

[17:50:51.242]: Unable to read VR Path Registry from C:\Users\Gokhan\AppData\Local\openvr\openvrpaths.vrpath
InitializeFileLogging failed

[17:50:51.436]: InitializeFileLogging failed
*** GPU Warning: Special file 'TempDisableGPU2' found -- skipping GPU2 ***
    name                                    : NVIDIA GeForce GTX 960M
    VendorId                                : 4318
    DeviceId                                : 5019
    SubSysId                                : 24122712
    Revision                                : 162
    DedicatedVideoMemory                    : 2107703296
    DedicatedSystemMemory                   : 0
    SharedSystemMemory                      : 8533104640
    MemoryBudget                            : 1791547801
    AdapterLuid.LowPart                     : 48051
    AdapterLuid.HighPart                    : 0
    Flags                                   : 0
        DoublePrecisionFloatShaderOps           : yes
        ConservativeRasterizationTier           : 0
        CrossNodeSharingTier                    : 0
        ResourceBindingTier                     : 3
        ResourceHeapTier                        : 1
        TiledResourcesTier                      : 1
        NodeIndex                               : 0
        TileBasedRenderer                       : no
        UMA                                     : no
        CacheCoherentUMA                        : no
        feature level                           : 0x00009100
        feature level                           : 0x00009200
        feature level                           : 0x00009300
        feature level                           : 0x0000a000
        feature level                           : 0x0000a100
        feature level                           : 0x0000b000
        feature level                           : 0x0000b100
        feature level                           : 0x0000c000
        feature level                           : 0x0000c100
        MaxSupportedFeatureLevel                : 0x0000b000
        HighestShaderModel                      : 0x60
        WaveOps                                 : yes
        WaveLaneCountMin                        : 32
        WaveLaneCountMax                        : 32
        TotalLaneCount                          : 640
        ExpandedComputeResourceStates           : yes
        Int64ShaderOps                          : yes
        RootSignature.HighestVersion            : 0x2
Skipping DX support for pre-Skylake Intel GPU.

[17:50:51.436]: InitializeFileLogging failed
*** GPU Warning: Special file 'TempDisableGPU2' found -- skipping GPU2 ***
    name                                    : NVIDIA GeForce GTX 960M
    VendorId                                : 4318
    DeviceId                                : 5019
    SubSysId                                : 24122712
    Revision                                : 162
    DedicatedVideoMemory                    : 2107703296
    DedicatedSystemMemory                   : 0
    SharedSystemMemory                      : 8533104640
    MemoryBudget                            : 1791547801
    AdapterLuid.LowPart                     : 48051
    AdapterLuid.HighPart                    : 0
    Flags                                   : 0
        DoublePrecisionFloatShaderOps           : yes
        ConservativeRasterizationTier           : 0
        CrossNodeSharingTier                    : 0
        ResourceBindingTier                     : 3
        ResourceHeapTier                        : 1
        TiledResourcesTier                      : 1
        NodeIndex                               : 0
        TileBasedRenderer                       : no
        UMA                                     : no
        CacheCoherentUMA                        : no
        feature level                           : 0x00009100
        feature level                           : 0x00009200
        feature level                           : 0x00009300
        feature level                           : 0x0000a000
        feature level                           : 0x0000a100
        feature level                           : 0x0000b000
        feature level                           : 0x0000b100
        feature level                           : 0x0000c000
        feature level                           : 0x0000c100
        MaxSupportedFeatureLevel                : 0x0000b000
        HighestShaderModel                      : 0x60
        WaveOps                                 : yes
        WaveLaneCountMin                        : 32
        WaveLaneCountMax                        : 32
        TotalLaneCount                          : 640
        ExpandedComputeResourceStates           : yes
        Int64ShaderOps                          : yes
        RootSignature.HighestVersion            : 0x2
Skipping DX support for pre-Skylake Intel GPU.

[17:50:56.934]: PROGRESS: Adding specified comp to Render Queue

[17:50:56.935]: PROGRESS: Adding specified comp to Render Queue

[17:50:57.064]: PROGRESS: Adding specified comp to Render Queue

[17:50:57.064]: PROGRESS: Adding specified comp to Render Queue

[17:50:57.693]: aerender ERROR: No render settings template was found with the given name.
PROGRESS:  11-Feb-21 17:50:57: Starting composition Main.



[17:50:57.694]: aerender ERROR: No render settings template was found with the given name.
PROGRESS:  11-Feb-21 17:50:57: Starting composition Main.



[17:50:58.527]: PROGRESS:  Render Settings: DuBest
PROGRESS:  Quality: Best
PROGRESS:  Resolution: Full
PROGRESS:  Size: 1280 x 720
PROGRESS:  Proxy Use: Use No Proxies
PROGRESS:  Effects: Current Settings
PROGRESS:  Disk Cache: Read Only
PROGRESS:  Color Depth: 8
PROGRESS:  Frame Blending: On for Checked Layers
PROGRESS:  Field Render: Off
PROGRESS:  Pulldown: Off
PROGRESS:  Motion Blur: On for Checked Layers
PROGRESS:  Solos: All Off
PROGRESS:  Time Span: Work Area
PROGRESS:  Start: 0:00:00:00
PROGRESS:  End: 0:00:02:25
PROGRESS:  Duration: 0:00:02:27
PROGRESS:  Frame Rate: 0.00 
PROGRESS:  Skip Existing Files: Off
PROGRESS:  
PROGRESS:  Output Module: DuWAV
PROGRESS:  Output To: C:\Users\Gokhan\AppData\Local\Temp\DuME - Duduf Media Encoder\aeCache\DuME_Cache.gDExNg\DuME.wav
PROGRESS:  Format: WAV
PROGRESS:  Output Info: -
PROGRESS:  Include: Project Link
PROGRESS:  Output Audio: 96.000 kHz / 32 bit F / Stereo
PROGRESS:  Channels: -
PROGRESS:  Depth: -
PROGRESS:  Color: -
PROGRESS:  Resize: -
PROGRESS:  Crop: -
PROGRESS:  Final Size: -
PROGRESS:  Profile: -
PROGRESS:  Post-Render Action: None
PROGRESS:  
PROGRESS:  0:00:00:00 (1): 0 Seconds
PROGRESS:  11-Feb-21 17:50:57: Finished composition Main.



PROGRESS:  Total Time Elapsed: 0 Seconds

[17:50:58.531]: PROGRESS:  Render Settings: DuBest
PROGRESS:  Quality: Best
PROGRESS:  Resolution: Full
PROGRESS:  Size: 1280 x 720
PROGRESS:  Proxy Use: Use No Proxies
PROGRESS:  Effects: Current Settings
PROGRESS:  Disk Cache: Read Only
PROGRESS:  Color Depth: 8
PROGRESS:  Frame Blending: On for Checked Layers
PROGRESS:  Field Render: Off
PROGRESS:  Pulldown: Off
PROGRESS:  Motion Blur: On for Checked Layers
PROGRESS:  Solos: All Off
PROGRESS:  Time Span: Work Area
PROGRESS:  Start: 0:00:00:00
PROGRESS:  End: 0:00:02:25
PROGRESS:  Duration: 0:00:02:27
PROGRESS:  Frame Rate: 0.00 
PROGRESS:  Skip Existing Files: Off
PROGRESS:  
PROGRESS:  Output Module: DuWAV
PROGRESS:  Output To: C:\Users\Gokhan\AppData\Local\Temp\DuME - Duduf Media Encoder\aeCache\DuME_Cache.gDExNg\DuME.wav
PROGRESS:  Format: WAV
PROGRESS:  Output Info: -
PROGRESS:  Include: Project Link
PROGRESS:  Output Audio: 96.000 kHz / 32 bit F / Stereo
PROGRESS:  Channels: -
PROGRESS:  Depth: -
PROGRESS:  Color: -
PROGRESS:  Resize: -
PROGRESS:  Crop: -
PROGRESS:  Final Size: -
PROGRESS:  Profile: -
PROGRESS:  Post-Render Action: None
PROGRESS:  
PROGRESS:  0:00:00:00 (1): 0 Seconds
PROGRESS:  11-Feb-21 17:50:57: Finished composition Main.



PROGRESS:  Total Time Elapsed: 0 Seconds

[17:50:59.375]: 
[17:50:59.376]: 
```

**Comment by Nico-Duduf** [comment]
Could you try with a more recent version of Ae?
for some reason the render settings Dume needs for rendering the frames are not loaded and I'd like to know if it's linked to the version in use.

Anyway, I can fix this when using the AE script to send the comp to Dume, will do in an update soon.

**Comment by Nico-Duduf** [comment]
You're on 15.1 and the latest version is 17.x, unfortunately we can't test and support versions that old due to install restrictions from Adobe

**Comment by JFMugen** [comment]
> Could you try with a more recent version of Ae?

Interesting, on AE 17.1.4x37, it worked first. Then it didn't work on the second time (sadly I didn't copy log this time since I was sure it will crash again) but after that, it worked every time. I tried to recreate "not working error" again, but haven't been able to. 

AE 15 still crashes. 

So, the crash is expected on the v15 I guess? I will share the log if I can somehow recreate the crash on v17, but looks like it will work even if I close it from the task manager this time :D

**Comment by Nico-Duduf** [comment]
I don't know for 15, looks like there is a faulty preset file...
I know a workaround anyway from the script in Ae I'll add in an update soon.

**Comment by JFMugen** [comment]
> I know a workaround anyway from the script in Ae I'll add in an update soon.

Great to hear. So far it is working on v17 relentlessly, I am not sure if I should be happy or sad because I can't make it reproduce :D

**Comment by Nico-Duduf** [comment]
This bug is fixed in the version which will be released in a few days (0.2.1) (but only when launching the render with the script from inside After Effects)
